config/application.rb

```diff
@@ -14,5 +14,7 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
```

Comment:
この一文があるので、ここで読み込む必要が無い場合には initializer に書いていました。（その方が良いと言っているのかと）
rails guides には、両方の例が載ってはいますが、どうなんでしょう。

* [Rails Internationalization (I18n) API — Ruby on Rails Guides](http://guides.rubyonrails.org/i18n.html)

2.1 Configure the I18n Module

> The load path must be specified before any translations are looked up. To change the default locale from an initializer instead of config/application.rb:

Reply:
なるほど、initializersに寄せるのが今風の書き方なのでしょうね。ありがとうございます。